src/coreclr/vm/i386/excepcpu.h

```diff
@@ -89,9 +89,9 @@ EXTERN_C LPVOID STDCALL COMPlusEndCatch(LPVOID ebp, DWORD ebx, DWORD edi, DWORD
 // RedirectedHandledJITCaseForXXX_Stub's.
 //
 PTR_CONTEXT GetCONTEXTFromRedirectedStubStackFrame(CONTEXT * pContext);
-#ifdef FEATURE_EH_FUNCLETS
+#if defined(FEATURE_EH_FUNCLETS) && !defined(TARGET_UNIX)
```

Comment:
Would it be possible to port Windows x86 to new EH in the future?

Reply:
Yes, it is something we have in our backlog

Reply:
I think it is better to delete this since there is no implementation.

This method has to more to do with low-level implementation of EH and thread suspension than funclets. If I am reading the code correctly, this method is only called from FixRedirectContextHandler that is only used on Windows x64. It can be probably deleted everywhere else as well.

Reply:
> It can be probably deleted everywhere else as well.

I do plan to delete it on UNIX. Including in this PR should be fine.
For Windows x86, leaving a stub may be useful for future implementation. Deleting it should also be fine because there should be more missing definitions for new EH on Windows x86.

Reply:
I do not think it will be needed for new EH on Windows x86

Reply:
It seems that `GetFrameFromRedirectedStubStackFrame` and more related code are not used for UNIX.
Before including more, I'd like to confirm that would we ever want them on UNIX?

Reply:
`DISPATCHER_CONTEXT` is a Windows SEH specific type. We try to stay away from building more Windows emulators for Unix, so I do not think that it will be ever needed on Unix.

I would leave FEF alone. I am not sure whether it is a good idea to try to eliminate it.

Reply:
FEF is still used in stack overflow handler. `DISPATCHER_CONTEXT` is also used by a bit more components. I think it's better to keep the scope of this PR.